.github/workflows/test-internal-prs.yml

```
@@ -111,7 +111,7 @@ jobs:
       repository: ${{ github.repository }}
 
   e2e:
-    uses: aws-amplify/amplify-ui/.github/workflows/reusable-e2e.yml@main
+    uses: aws-amplify/amplify-ui/.github/workflows/reusable-e2e.yml@move-liveness-e2e-account
```

Let's get ready to revert this pre-merge 😈 
```suggestion
    uses: aws-amplify/amplify-ui/.github/workflows/reusable-e2e.yml@main
```